src/Serilog.Sinks.RavenDB/Sinks/RavenDB/RavenDBSink.cs

```diff
@@ -47,12 +52,26 @@ public class RavenDBSink : PeriodicBatchingSink
         /// <param name="batchPostingLimit">The maximum number of events to post in a single batch.</param>
         /// <param name="period">The time to wait between checking for event batches.</param>
         /// <param name="formatProvider">Supplies culture-specific formatting information, or null.</param>
-        public RavenDBSink(IDocumentStore documentStore, int batchPostingLimit, TimeSpan period, IFormatProvider formatProvider)
+        /// <param name="defaultDatabase">Optional name of default database</param>
+        /// <param name="expirationTimeSpan">Optional time before a logged message will be expired assuming the expiration bundle is installed. Zero (00:00:00) means no expiration. If this is not provided but errorExpirationTimeSpan is, errorExpirationTimeSpan will be used for non-errors too.</param>
```

Comment:
Yeah, I thought about InfiniteTimeout too but was looking for something that would be easy to represent in a config file (as I am doing where I first used it); "00:00:00" is easier than "-00:00:00.0010000".  Given the philosophy of serilog, which seems to be focused on configuration in code, I'm changing it as you suggest. I documented the heck out of it so hopefully it won't be too hard to use in a config file.